Description: Enforce python3 shebang line for Python 2+3 compatible userscripts
Author: Axel Beckert <abe@debian.org>
Forwarded: not-needed

--- a/misc/userscripts/readability
+++ b/misc/userscripts/readability
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Executes python-readability on current page and opens the summary as new tab.
 #
--- a/misc/userscripts/ripbang
+++ b/misc/userscripts/ripbang
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Adds DuckDuckGo bang as searchengine.
 #
